core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -33,6 +32,12 @@ type l1OracleClient interface {
 	BatchCallContext(ctx context.Context, b []rpc.BatchElem) error
 }
 
+// Client is interface of client connections for additional chains layers
+type Client interface {
```

Comment:
IMO Client is too generic of a name for this purpose. Perhaps something like `DAClient` or something similar is better.